src/pages/user/UserRoles.tsx

```diff
@@ -2,44 +2,39 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 
 import IconLabelTextLine from '../../components/IconLabelTextLine';
-import { RoleName, User } from '../../types/user.types';
+import { User } from '../../types/user.types';
 import Card from '../../components/Card';
 import Heading from '../../components/Heading';
+import { checkIfPublisher, checkIfCurator } from '../../utils/authorization';
 
 interface UserRolesProps {
   user: User;
 }
 
 const UserRoles: React.FC<UserRolesProps> = ({ user }) => {
   const { t } = useTranslation('profile');
+  const isPublisher = checkIfPublisher(user);
+  const isCurator = checkIfCurator(user);
 
   return (
     <Card>
       <Heading>{t('heading.roles')}</Heading>
-      {user.roles?.map((role: RoleName) => {
-        if (role === RoleName.PUBLISHER) {
-          return (
-            <IconLabelTextLine
-              dataTestId="user-role"
-              icon={'create'}
-              label={t('roles.publisher')}
-              text={t('roles.publisher_description')}
-              key={role}
-            />
-          );
-        } else if (role === RoleName.CURATOR) {
-          return (
-            <IconLabelTextLine
-              dataTestId="user-role"
-              icon="all_inbox"
-              label={t('roles.curator')}
-              text={t('roles.curator_description')}
-              key={role}
-            />
-          );
-        }
-        return null;
-      })}
+      {isPublisher && (
+        <IconLabelTextLine
+          dataTestId="user-role"
+          icon={'create'}
+          label={t('roles.publisher')}
+          text={t('roles.publisher_description')}
+        />
+      )}
+      {isCurator && (
+        <IconLabelTextLine
+          dataTestId="user-role"
+          icon="all_inbox"
```

Comment:
hva er forskjellen på `icon={"create"}` og `icon="all_inbox"` -- altså med og uten wrapping av krøllparanteser?

Reply:
krøllparantes er optional for strings

Reply:
kan fjerne for create også